pydyad/setup.cfg

```diff
@@ -8,4 +8,5 @@ classifier =
 python_requires = >=3.7
 install_requires =
     numpy
-    dlio_profiler_py @ git+https://github.com/hariharan-devarajan/dlio-profiler.git
\ No newline at end of file
+    # dlio_profiler_py @ git+https://github.com/hariharan-devarajan/dlio-profiler.git
+    pydftracer @ git+https://github.com/hariharan-devarajan/dftracer.git
```

Comment:
Do u want to switch it to pydftrace==1.0.2. Its better than a moving target.

Reply:
I didn't realize that you had pinned a version. I'll switch to that.